### PR TITLE
Add Occurrence.groupSize to standard Standard Excel Export

### PR DESCRIPTION
**Pull request by holmbergius**
User request to add Occurrence.groupSize to Standard Excel Export.

**Changes**
- Adds a single line of code to enable Occurrence.groupSize field export in Excel

### CODE REVIEW

**Review by TanyaStere42 — CHANGES_REQUESTED** [review]
since this creates the columns in order of appearance, can you move this to either the beginning or the end of the group size estimates?

File: src/main/java/org/ecocean/servlet/export/EncounterSearchExportMetadataExcel.java [hunk 1]
```diff
@@ -171,6 +171,7 @@ public void doPost(HttpServletRequest request, HttpServletResponse response)
             newEasyColumn("Occurrence.groupBehavior", columns);
             newEasyColumn("Occurrence.minGroupSizeEstimate", columns);
             newEasyColumn("Occurrence.bestGroupSizeEstimate", columns);
+            newEasyColumn("Occurrence.groupSize", columns);
```
Comment (naknomum) [inline src/main/java/org/ecocean/servlet/export/EncounterSearchExportMetadataExcel.java:174]: is there a `groupSize` on Occurrence? maybe i am missing it... i see a method for calculating an estimate one. and the above properties like `bestGroupSizeEstimate`.  could this be specific to only _some_ wildbooks?

Comment (holmbergius) [reply src/main/java/org/ecocean/servlet/export/EncounterSearchExportMetadataExcel.java:174]: This ended up being a GiraffeSpotter only field. Nice catch @naknomum! I will work to convert them over to the existing, standardized field of Occurrence.bestGroupSizeEstimate

Comment (holmbergius) [reply src/main/java/org/ecocean/servlet/export/EncounterSearchExportMetadataExcel.java:174]: Closing PR without merge as a result.

**Pull request closed without merging**